modules/ibi-create-standalone-config-iso.adoc

```diff
@@ -0,0 +1,229 @@
+# See https://nmstate.io/examples.html for examples.
+networkConfig:
+  interfaces:
+    - name: eth0
+      type: ethernet
+      state: up
+      #mtu: 9000 <1>
+      mac-address: 00:00:00:00:00:00
+      ipv4:
+        enabled: true
+        address:
+          - ip: 192.168.122.2
+            prefix-length: 23
+        dhcp: false
+----
+<1> If you configure a maximum transmission unit (MTU) in the seed cluster, you must apply the same MTU value in the static network configuration for the configuration ISO.
+
+. Configure your configuration file:
++
+.Example `image-based-config.yaml` file
+[source,yaml]
+----
+#
+# Note: This is a sample ImageBasedConfig file showing
+# which fields are available to aid you in creating your
+# own image-based-config.yaml file.
+#
+apiVersion: v1beta1
+kind: ImageBasedConfig
+metadata:
+  name: sno-cluster-name
+additionalNTPSources:
+  - 0.rhel.pool.ntp.org
+  - 1.rhel.pool.ntp.org
+hostname: host.example.com
+releaseRegistry: quay.io
+# networkConfig contains the network configuration for the host in NMState format.
+# See https://nmstate.io/examples.html for examples.
+networkConfig:
+    interfaces:
+      - name: ens1f0
+        type: ethernet
+        state: up
+        ipv4:
+          enabled: true
+          dhcp: false
+          auto-dns: false
+          address:
+            - ip: 10.6.85.8
+              prefix-length: 24
+        ipv6:
+          enabled: false
+    dns-resolver:
+      config:
+        server:
+          - 10.6.73.2
+          - 10.6.73.4
+    routes:
+      config:
+      - destination: 0.0.0.0/0
+        metric: 150
+        next-hop-address: 10.6.85.254
+        next-hop-interface: ens1f0
+----
+
+. Create the configuration ISO in your working directory by running the following command:
++
+[source,terminal]
+----
+$ openshift-install image-based create config-image --dir ibi-config-iso-workdir/
+----
++
+.Example output
+[source,terminal]
+----
+INFO Adding NMConnection file <ens1f0.nmconnection> 
+INFO Consuming Install Config from target directory 
+INFO Consuming Image-based Config ISO configuration from target directory 
+INFO Config-Image created in: ibi-config-iso-workdir/auth
+----
++
+View the output in the working directory:
++
+.Example output
+[source,terminal]
+----
+ibi-config-iso-workdir/
+├── auth
+│   ├── kubeadmin-password
+│   └── kubeconfig
+└── imagebasedconfig.iso
+----
+
+. Attach the `imagebasedconfig.iso` to the preinstalled host using your preferred method and restart the host to complete the configuration process and deploy the cluster.
+
+.Verification
+When the configuration process completes on the host, access the cluster to verify its status.
+
+. Export the `kubeconfig` environment variable to your kubeconfig file by running the following command:
++
+[source,terminal]
+----
+$ export KUBECONFIG=ibi-config-iso-workdir/auth/kubeconfig
+----
+
+. Verify the cluster is ready by running the following command:
```

Comment:
```suggestion
. Verify the cluster is responding by running the following command:
```

Reply:
Updated. "Verify that the cluster is responding by the following command:"